### ISSUE TITLE
Anomaly on application_start duration

### ISSUE THREAD

**Issue by fahmisdk6** [DataDog/dd-sdk-flutter#537]
**Describe what happened**
We have over 1k sessions with `application_start` duration of over 1 hour. But the user behavior seems normal. Is it a known bug for `dd-sdk-flutter` on Android OS?  Is there a way to fix it?

**Steps to reproduce the issue:**
Search actions using this query
`@action.type:application_start @session.type:user env:release @os.name:Android `

<img width="1345" alt="Screen Shot 2024-01-04 at 17 18 35" src="https://github.com/DataDog/dd-sdk-flutter/assets/10940190/7cff58d6-f107-4c8c-a165-72fbc0f19549">

User behavior seems fine even with the 2-hour-long startup time:

<img width="877" alt="Screen Shot 2024-01-04 at 17 25 19" src="https://github.com/DataDog/dd-sdk-flutter/assets/10940190/589fb208-38f2-4532-bf2c-cebd198688a2">

**Describe what you expected:**

application_start working normally, without anomaly of duration > 1 hour

**Additional context**
 - Flutter version: 3.7.12
 - Android: across all versions

**Comment by fuzzybinary** [comment]
Hi @fahmisdk6, thanks for reporting.

This looks related to https://github.com/DataDog/dd-sdk-android/issues/1765

What version of the Datadog SDK are you using?

**Comment by fahmisdk6** [comment]
ah we are still using v1.6.2. we'll update to v2.x then

**Comment by fuzzybinary** [comment]
Be aware this may not fix the issue, we're still investigating.

To help us, can you let us know if your app does any background work?

**Comment by fahmisdk6** [comment]
Yes we are aware, thanks.

No our app does not have any background work

**Comment by fuzzybinary** [comment]
Hi @fahmisdk6,

Are you still seeing this issue?

**Comment by fahmisdk6** [comment]
Yes @fuzzybinary, unfortunately upgrading the dd version does not solve it.

**Comment by fuzzybinary** [comment]
Okay, thanks for letting us know.  We'll continue to look into it.

Is there anything you can tell us about your setup that might be unusual?
